repository/provider_queries.py

```diff
@@ -20,3 +24,12 @@ def get_enabled_providers_for_pro() -> List[Provider]:
         .filter_by(enabledForPro=True) \
         .order_by(Provider.name) \
         .all()
+
+
+def get_enabled_providers_for_pro_excluding_provider(allocine_local_class: str) -> List[Provider]:
```

Comment:
pas hyper clair comme nom.

```suggestion
def get_providers_enabled_for_pro_excluding_a_specific_provider(allocine_local_class: str) -> List[Provider]:
```

Reply:
Yes !